repository/c.json

```diff
@@ -415,6 +415,18 @@
 				}
 			]
 		},
+		{
+			"name": "Catppuccin color schemes", 
+			"details": "https://github.com/catppuccin/sublime-text",
+			"author": ["Catppuccin"],
+			"labels": ["color scheme"],
+			"releases": [
+				{
+					"sublime_text": "*",
```

Comment:
This should be `"sublime_text" : ">=3100"` because `.sublime-color-scheme` support was added in ST 3.1